common/lib/container-definitions/api-report/container-definitions.api.md

```diff
@@ -56,12 +56,15 @@ export enum BindState {
 // @public
 export namespace ConnectionState {
     export type Connected = 2;
+    // @deprecated (undocumented)
     export type Connecting = 1;
     export type Disconnected = 0;
+    export type Pending = 1;
```

Comment:
Maybe "CatchingUp"? Pending does not reflect actual state

Reply:
We'll it also covers corner cases like where previous Leave op is dropped, so I thought a more generic term made sense. Totally open minded though.